sport/app/football/views/fragments/matchSummary.scala.html

```diff
@@ -35,67 +36,47 @@ <h1 class="u-h">@homeTeam.name v @awayTeam.name</h1>
     </div>
 
     <div class="match-info">
-        @competition.map{ competition => <div class="match-info__competition">@competition.fullName</div>}
         @theMatch.venue.map{ venue => <div class="match-info__venue">@venue.name</div> }
     </div>
+-->
 
-    <div class="match-summary__teams">
-        <div class="match-summary__team match-summary__team--home u-cf">
-            <div class="team__crest team__crest--home">
-                <div class="team__crest__img-container">
-                    <span class="team__crest__img" style="background-image: url(@Configuration.staticSport.path/football/crests/120/@{homeTeam.id}.png)"></span>
-                </div>
-            </div>
-            <div class="team__info">
-                <h2 class="team__heading">
-                    <span class="team__name" data-abbr="@pa.TeamCodes.codeFor(homeTeam)">
-                        <span class="team__name-long">@homeTeam.name</span>
-                    </span>
-                    @if(theMatch.isLive || theMatch.isResult){<span class="team__score">@homeTeam.score</span>}
-                </h2>
+    <div class='match-summary__teams'>
```

Comment:
I'm not sure why code style isn't enforced in Play templates, but can we replace the single quotes around HTML attributes with double quotes?